apps/web/components/EthAddress.tsx

```diff
@@ -35,7 +35,7 @@ export const EthAddress = ({
   //   popupRadius: 12,
   //   popupShadow: "black",
   // };
-  const chainId = getChainIdFromPath();
+  const chainId = getChainIdFromPath() as number;
```

Comment:
the function say return type its chainID = string | number, but always convert to number, so should be unecessary and chainID changed or just return number in that func

Reply:
https://github.com/1Hive/gardens-v2/pull/229/commits/707108277c131da4cb3496f05337c09540390770

Reply:
https://github.com/1Hive/gardens-v2/pull/229/commits/5496912036e05bbb3274af4d9c822c308e3b2946